google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/ProjectDebuggeeBinding.java

```diff
@@ -48,24 +49,28 @@
  */
 class ProjectDebuggeeBinding {
   private static final Logger LOG = Logger.getInstance(ProjectDebuggeeBinding.class);
-  private final JComboBox myDebugeeTarget;
-  private final ProjectSelector myElysiumProjectId;
-  private Debugger myCloudDebuggerClient = null;
-  private CredentialedUser myCredentialedUser = null;
-  private CloudDebugProcessState myInputState;
-
-  public ProjectDebuggeeBinding(@NotNull ProjectSelector elysiumProjectId, @NotNull JComboBox debugeeTarget) {
-    myElysiumProjectId = elysiumProjectId;
-    myDebugeeTarget = debugeeTarget;
-
-    myElysiumProjectId.getDocument().addDocumentListener(new DocumentAdapter() {
+  private final JComboBox moduleSelector;
+  private final ProjectSelector elysiumProjectId;
```

Comment:
Definitely. Missed that one. Its been changed as well as minor naming cleanup in DebugTarget to make things more consistent (removing the "my" prefix from fields).